packages/abstractions/src/multipartBody.ts

```diff
@@ -93,7 +93,7 @@ interface MultipartEntry {
 	serializationCallback?: ModelSerializerFunction<Parsable>;
 }
 
-export const serializeMultipartBody = (writer: SerializationWriter, multipartBody: Partial<MultipartBody> | undefined | null = new MultipartBody()): void => {
```

Comment:
please revert the const changes on this line. Here is more context https://www.freecodecamp.org/news/the-difference-between-arrow-functions-and-normal-functions/

Reply:
![removed](https://github.com/user-attachments/assets/481d0f97-47e2-4dfc-bfdb-416921235ad6)

Isn't this what we moved from?